examples/implied-original.diag

```diff
@@ -1,22 +1 @@
-[
-  
-  h'd3c14744d1791d02548232c23d35efa97668174ba385af066011e43bd7e51501',
-  
-  
-  62(1644387225019),
```

Comment:
Timestamps still have the 62 tag, where does this number come from?

Reply:
It is a CBOR tag for timestamps. I can remove this or expand the explanation. This really should depend on discussing the timestamp granularity in mimi-protocol first.

Reply:
Tracked in issue #13